### PR TITLE
fix: stop checking folder permissions (main)

### PR DESCRIPTION
**Pull request by martabal**
Stop checking folder permissions, because it can take a looooong time

### CODE REVIEW

File: root/etc/s6-overlay/s6-rc.d/init-config-immich/run [hunk 1]
```diff
@@ -7,11 +7,14 @@ mkdir -p \
     /import \
     /run/immich
 
+# check if /photos is empty
```
Comment (hydazz) [inline root/etc/s6-overlay/s6-rc.d/init-config-immich/run:10]: why bother if the folder is empty?

Comment (martabal) [reply root/etc/s6-overlay/s6-rc.d/init-config-immich/run:10]: Otherwise, the CI fails

Comment (hydazz) [reply root/etc/s6-overlay/s6-rc.d/init-config-immich/run]: > Otherwise, the CI fails

with what error?

Comment (martabal) [reply root/etc/s6-overlay/s6-rc.d/init-config-immich/run]: je n'arrive pas à retrouver l'erreur :thinking:

Comment (hydazz) [reply root/etc/s6-overlay/s6-rc.d/init-config-immich/run]: really make me whip out the translator

Comment (martabal) [reply root/etc/s6-overlay/s6-rc.d/init-config-immich/run]: Lol, "can't find the error"

Comment (martabal) [reply root/etc/s6-overlay/s6-rc.d/init-config-immich/run]: Here you go `Error: EACCES: permission denied, mkdir '/photos/library' `

Comment (hydazz) [reply root/etc/s6-overlay/s6-rc.d/init-config-immich/run]: thats /photos not /import?

Comment (hydazz) [reply root/etc/s6-overlay/s6-rc.d/init-config-immich/run]: `find /photos -maxdepth 1 -type d -exec chown abc:abc {} \;`

Comment (martabal) [reply root/etc/s6-overlay/s6-rc.d/init-config-immich/run]: Yep, the issue is `/photos`. Someone reported a startup time which took 1h (he has 400 GB of assets)

**Review by hydazz — APPROVED** [review]

**Comment by martabal** [comment]
@hydazz All PRs should be good

**Pull request merged**